timemachine/fe/rbfe.py

```diff
@@ -738,7 +749,7 @@ def run_edge_and_save_results(
         return path
 
     path = f"success_rbfe_result_{edge.mol_a_name}_{edge.mol_b_name}.pkl"
-    pkl_obj = (mol_a, mol_b, edge.metadata, smarts, core, solvent_res, solvent_top, complex_res, complex_top)
```

Comment:
@jkausrelay sanity check: does this affect anything on the consumer end?

Reply:
This would only affect @mcwitt , we call the run_* directly